playground/src/pages/CodingEnvironment.tsx

```diff
@@ -87,18 +89,21 @@ export default function CodingEnvironment(): JSX.Element {
   const runProgram = async () => {
     init().then(() => {
       disableDebug();
+      setProof(null);
       const inputCheck = checkInputs(inputs);
       if (!inputCheck.isValid) {
         setOutput(inputCheck.errorMessage);
         toast.error("Execution failed");
         return;
       }
       try {
+        const start = Date.now();
         const { stack_output, trace_len }: Outputs = run_program(code, inputs);
         setOutput(`{
 "stack_output" : [${stack_output.toString()}],
 "trace_len" : ${trace_len}
 }`);
+      toast.success(`Execution successful in ${Date.now() - start} ms`);
```

Comment:
this is oddly indented, please double check you're applying formatting correctly

Reply:
applied the formatter